db/schema.rb

```diff
@@ -12,7 +12,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190318223433) do
+ActiveRecord::Schema.define(version: 20190326085046) do
```

Comment:
Looks like this is missing the index in the schema itself?

Reply:
Thanks, fixed!